runtime/README.rst

```diff
@@ -3,35 +3,95 @@
+    contains implementations of the ``QuantumDevice`` API for backend simulators.
+
+- `tests <https://github.com/PennyLaneAI/catalyst/tree/main/runtime/tests>`_:
+    A collection of C++ tests for modules and methods in the runtime.
+
+Backend Devices
+===============
+
+A new device can be added to the list of supported ones in the runtime by implementing the quantum device API.
+The following table shows the runtime official backend devices along with supported features:
+
+.. list-table::
+   :widths: 25 25 25
+   :header-rows: 0
+
+   * - **Features**
+     - **PennyLane-Lightning**
+     - **PennyLane-Lightning-Kokkos**
+   * - Qubit Management
+     - Dynamic allocation/deallocation
+     - Static allocation/deallocation
+   * - Gate Operations
+     - `Lightning operations <https://github.com/PennyLaneAI/pennylane-lightning/blob/master/pennylane_lightning/src/gates/GateOperation.hpp>`_
+     - `Lightning operations <https://github.com/PennyLaneAI/pennylane-lightning/blob/master/pennylane_lightning/src/gates/GateOperation.hpp>`_
+   * - Quantum Observables
+     - ``Identity``, ``PauliX``, ``PauliY``, ``PauliZ``, ``Hadamard``, ``Hermitian``, ``Hamiltonian``, Tensor Product of Observables
+     - ``Identity``, ``PauliX``, ``PauliY``, ``PauliZ``, ``Hadamard``, ``Hermitian``, ``Hamiltonian``, Tensor Product of Observables
+   * - Expectation Value
+     - All observables
+     - Basic observables (``Identity``, ``PauliX``, ``PauliY``, ``PauliZ``, ``Hadamard``, ``Hermitian``)
+   * - Variance
+     - ``Identity``, ``PauliX``, ``PauliY``, ``PauliZ``
+     - Not supported
+   * - Finite Shots (``Sample``)
+     - Only for the computational basis on the supplied qubits
+     - Only for the computational basis on the supplied qubits
+   * - Probabilities
+     - Only for the computational basis on the supplied qubits
+     - Only for the computational basis on the supplied qubits
+   * - Mid-Circuit Measurement
+     - Only for the computational basis on the supplied qubit
+     - Only for the computational basis on the supplied qubit
+   * - Gradients
+     - The Adjoint-Jacobian method supporting all observables
+     - The Adjoint-Jacobian method supporting all observables
 
 Requirements
 ============
 
-To build the runtime from source, it is required to have an up to date version of a C/C++ compiler such as gcc, clang, or MSVC and the static library of
-``stdlib`` from `qir-runner <https://github.com/qir-alliance/qir-runner/tree/main/stdlib>`_.
-
-The ``make runtime`` command uses ``clang`` and ``clang++`` as the default C and C++ compilers. You can use ``c_compiler`` and ``cpp_compiler`` make
-options to change the default behavior.
-
-.. code-block:: console
-
-    make runtime c_compiler=$(which gcc) cpp_compiler=$(which g++)
+To build the runtime from source, it is required to have an up to date version of a C/C++ compiler such as gcc, clang, or MSVC
```

Comment:
Until we have Windows support it might be confusing to list MSVC
```suggestion
To build the runtime from source, it is required to have an up to date version of a C/C++ compiler such as gcc or clang
```